packpack

```diff
@@ -49,7 +49,7 @@ usage() {
     echo ""
     echo " * PRODUCT - the name of software product, e.g. 'tarantool'"
     echo " * OS - the name of distribution, e.g. debian or fedora (${OS})"
-    echi " * DIST - the version of distrition, e.g. xenial or 24 (${DIST})"
+    echo " * DIST - the version of distrition, e.g. xenial or 24 (${DIST})"
```

Comment:
Maybe fix the `distrition` typo too while you're at it? IMHO this would be better written as:
```shell
     echo " * PRODUCT - software product name, e.g. 'tarantool'"
     echo " * OS - distribution name, e.g. debian or fedora (${OS})"
     echo " * DIST - distribution version, e.g. xenial or 24 (${DIST})" 
```